core/trino-main/src/test/java/io/trino/execution/TestCommentTask.java

```diff
@@ -106,6 +107,22 @@ public void testCommentViewOnMaterializedView()
                 .hasMessage("View '%s' does not exist, but a materialized view with that name exists. Setting comments on materialized views is unsupported.", materializedViewName);
     }
 
+    @Test
+    public void testCommentTableColumn()
+    {
+        QualifiedObjectName tableName = qualifiedObjectName("existing_table");
+        QualifiedName columnName = qualifiedColumnName("existing_table", "test");
+        metadata.createTable(testSession, TEST_CATALOG_NAME, someTable(tableName), false);
+        assertThat(metadata.getTableMetadata(testSession, metadata.getTableHandle(testSession, tableName).get()).getMetadata().getComment())
+                .isEmpty();
```

Comment:
We can remove this assertion as the test purpose is verifying column comment, not table comment.